back-end/e-learning/app/Http/Controllers/CategoryController.php

```diff
@@ -37,4 +37,26 @@ public function list()
 	{
 		return response(Category::all(), 201);
 	}
+
+	public function update(Request $request, $id)
+	{
+		if(is_null($request->title) || is_null($request->description))
+		{
+			return response([
+				'message'=> 'Data is required.'
+			], 400);
+		}
+		$category = Category::find($id);
+		if($category){
```

Comment:
If youre gonna write another block of code after a block of code, make sure to put line space so that it's easy to spot the separation

```suggestion

		$category = Category::find($id);
		if ($category){
```